src/pages/RepoPage/RepoPage.tsx

```diff
@@ -245,6 +246,27 @@ function RepoPage() {
       refetchOnWindowFocus: refetchEnabled,
     },
   })
+  const { setBaseCrumbs } = useCrumbs()
```

Comment:
We don't have the Repo until the RepoPage is visited, so we populate the repo in the context only once we get here.